source/playground-databases.txt

```diff
@@ -0,0 +1,108 @@
+.. _vsce-playground-databases:
+
+================================
+Creating Databases & Collections
+================================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. include:: /includes/fact-vsce-preview.rst
+
+You can create 
+:manual:`databases and collections </core/databases-and-collections/>` 
+using a MongoDB Playground. 
+
+Prerequisites
+-------------
+
+If you have not done so already, you must complete the following 
+prerequisites before you can create a time series collection with a 
+MongoDB Playground:
+
+- :ref:`Create a connection to a MongoDB deployment 
+  <vsce-connect-task>`.
+- :ref:`Activate the connection to the MongoDB deployment 
+  <vsce-activate-connection>`.
+
+Create a Database and Collection
+--------------------------------
+
+Once you connect to your deployment using MongoDB for VS code, use the left 
+navigation to: 
+
+1. Select an active connection and click the :icon-fa5:`plus` icon that 
+   appears.
+   
+#. A MongoDB playground automatically opens with a template form to create 
+   a database and a regular collection or a 
+   :ref:`time series collection <vsce-time-series>`. 
+
+#. Fill in the provided fields with the name of your database and collection.
+
+#. Uncomment the form of the collection you wish to create and fill in the 
+   provided fields accordingly.
```

Comment:
[suggestion]

These steps feel slightly out of order / confusing to me since it kind of seems like we're telling users to fill in their collection info twice. Maybe we do something like:

```
# Uncomment the form of the collection you wish to create.
# Fill in the collection fields according to your collection's specifications.
```

Reply:
Done.